blocks/generator.go

```diff
@@ -60,6 +63,29 @@ func defaultConfig() Config {
 	}
 }
 
+type stats struct {
+	lock       sync.Mutex
```

Comment:
mu is more common

Reply:
thanks. noted. moved the code to certifier.go and reuse the mutex there.